run.apls

```diff
@@ -2,48 +2,74 @@
 
 ⍝ Full session start-up
 DYALOG ← 2 ⎕NQ #'GetEnvironment' 'DYALOG'
-⍎⊃2⎕FIX 'file://',DYALOG,'/startup.dyalog'
-
-⎕SE.Link.Import # '/Users/rush/wdr/projects/bb-test-apl'
-
-⍝ list of available tests
-ts←⊂'add'
-ts←⊂'divide'
-ts←⊂'floor'
-ts,←⊂'indexof'
-ts,←⊂'magnitude'
-ts,←⊂'membership'
-ts,←⊂'residue'
-ts←⊂'subtract'
-ts,←⊂'unique'
-ts←⊂'uniquemask'
-
-⎕←''
-⎕←'Hi, Choose supported tests:'
-⎕←'all', ts
-⎕←''
-⎕←'Enter TestName:'
-tName←¯1∘⎕C ⍞ ⍝ stdin to lowercase
-⎕←''
-
-⎕←''
-⎕←'Test options: (not supported with all, stop has limited functionality)'
-⎕←'verbose stop repetitions randLink'
-opts←⍎⍞
-⎕←opts
-(prod verbose stop reps randLink)←5↑opts,0 0 0 0 0
-⎕←''
-
-t←{
-    3::'Test not found, please select from the list'⎕SIGNAL 11
-    tIndex ←ts⍳⊂tName
-    tName≡'all':(⊂'tests.'),¨ts
-    'tests.',⊃ts[tIndex]
-}⍬
-
-run←{
-    tName≡'all' : unittest.RunTests ¨(⍎¨t)
-    unittest.RunTests (⍎t) verbose stop reps randLink
+⍎⊃2⎕FIX 'file://',DYALOG,'/StartupSession.aplf'

```

Comment:
```suggestion
⎕SE.(⍎⊃2⎕FIX'/StartupSession.aplf',⍨2⎕NQ#'GetEnvironment' 'DYALOG')
```

Reply:
Can you check if this works? Got this from aplcart while trying to fix my dyalogscript installatio